include/modem/location.h

```diff
@@ -82,7 +82,12 @@ enum location_event_id {
 	 * is available. The application should send the information to cloud services and
 	 * then call location_cloud_location_ext_result_set().
 	 */
-	LOCATION_EVT_CLOUD_LOCATION_EXT_REQUEST
+	LOCATION_EVT_CLOUD_LOCATION_EXT_REQUEST,
+	/**
+	 * Location rewquest has been started. This event is only sent if
+	 * @kconfig{CONFIG_LOCATION_DATA_DETAILS} is set.
```

Comment:
What do you think about separating some of descriptions into brief and detailed sections? I think it makes sense for example here.
```suggestion
	 * Location request has been started.
	 *
	 * This event is only sent if @kconfig{CONFIG_LOCATION_DATA_DETAILS} is set.
```

Reply:
I did it into this structure and few other places.